gensim/models/keyedvectors.py

```diff
@@ -1063,6 +1067,134 @@ def get_keras_embedding(self, train_embeddings=False):
         )
         return layer
 
+    def simple_average(self, sent):
+        """
+        Mixture of word-vectors to make sentence-vector.
+
+        Attains this by simple averaging of all word-vectors.
+
+        Examples
+        --------
```

Comment:
also, please write all imports directly in example (this should works if I copy-paste it to console)

Reply:
still not working (for all examples)